### DISCUSSION TITLE
Can we capture multiple photos at a time by using ImagePicker.launchCamera

### DISCUSSION THREAD

**Discussion by Yandamuri** [react-native-image-picker/react-native-image-picker#2061] [Q&A]
Following is the response of `ImagePicker.launchCamera()`,

<img width="913" alt="Screenshot 2022-12-09 at 3 31 04 PM" src="https://user-images.githubusercontent.com/31983789/206677041-10b95092-e4b3-4887-8ebf-4753690aba15.png">

And this is how the response object is used in example as per docs,

```
{response?.assets &&
          response?.assets.map(({uri}: {uri: string}) => (
            <View key={uri} style={styles.imageContainer}>
              <Image
                resizeMode="cover"
                resizeMethod="scale"
                style={styles.image}
                source={{uri: uri}}
              />
            </View>
          ))}
```

which means response contains an object which contains an array.

1. Are there any chances that objects contains multiple arrays?
2. Does response contains `assets` object always?

**Comment by alessandrom** [comment]
Anybody managed to capture multiple photos?
